### DISCUSSION TITLE
validation loss logging in validation_step

### DISCUSSION THREAD

**Discussion by yiftachbeer** [Lightning-AI/pytorch-lightning#15715] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hi,
I've noticed that the official tutorials that include both a [training step](https://pytorch-lightning.readthedocs.io/en/stable/model/train_model_basic.html) and a [validation step](https://pytorch-lightning.readthedocs.io/en/latest/common/evaluation_basic.html), **return** the training loss in training_step but **log** the validation loss in validation loss.

The reason for this difference is not documented. Does it not make sense for validation step to have the validation loss returned, instead of logged, as well?
